### ISSUE TITLE
Why not provide the simple tutorial in README?

### ISSUE THREAD

**Issue by begeekmyfriend** [sotelo/parrot#5]
As a green hand in Python and Theano, I want to know the command line to run the demo training the network according to the data set such as `VCTK` and generating the audio files. It is just a job of several line.

**Comment by manugarri** [comment]
@begeekmyfriend It seems that the authors have abandoned the project.

**Comment by sotelo** [comment]
Not really. We have just been super busy searching good configurations for our model. Also, we are waiting a bit on the results of our paper submission.

**Comment by manugarri** [comment]
@sotelo glad to hear that :) , are there plans to maintain/update this repo?

**Comment by sotelo** [comment]
Yes. Please have in mind that the ICLR paper was kind of preliminary results. We didn't do any hyperparameter search. We didn't explore architecture variants and so on...

Furthermore, there have been super interesting advances recently:
https://google.github.io/tacotron/

So we are updating/improving our model based on these results. Because of all this, we have not had as much time to clean the data processing code (which is the biggest barrier for people using this code). Furthermore, if we end up by changing the interface to the models when refactoring the code, it doesn't really make a lot of sense to spend some time writing a tutorial on how to use the current state of the code (mainly because most people won't be able to use it because of the data processing).

**Comment by manugarri** [comment]
I understand. Thanks for your reply, keep doing such a good job :)

**Comment by jdbermeol** [comment]
Hi @sotelo, I'm also interested in this model, what are your plans for this repo?

**Comment by gcunhase** [comment]
Any updates on a simple tutorial in the README file?
